nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSSequenceFile.java

```diff
@@ -108,6 +109,8 @@ protected void processBatchOfFiles(final List<Path> files, final ProcessContext
                 if (!keepSourceFiles && !hdfs.delete(file, false)) {
                     logger.warn("Unable to delete path " + file.toString() + " from HDFS.  Will likely be picked up over and over...");
                 }
+            } catch (final IOException e) {
```

Comment:
This way we don't handle non-GSS IOExceptions properly.

Instead of
```java
           } catch (final IOException e) {
                handleAuthErrors(e, session, context);
            } catch (Throwable t) {
                logger.error("Error retrieving file {} from HDFS due to {}", new Object[]{file, t});
                session.rollback();
                context.yield();
```
we should have
```java
            } catch (Throwable t) {
                if (!handleAuthErrors(t, session, context)) {
                    logger.error("Error retrieving file {} from HDFS due to {}", new Object[]{file, t});
                    session.rollback();
                    context.yield();
                }
```

Reply:
I think in the default exception handling the 

```java
session.rollback();
context.yield();
```
was removed.